test/cli.mjs

```diff
@@ -84,13 +84,42 @@ describe("postject CLI", () => {
       expect(status).to.equal(0);
     }
 
+    // Verifying code signing using a self-signed certificate.
+    {
+      if (process.platform === "darwin") {
+        let codesignFound = false;
+        try {
+          execSync("command -v codesign");
+          codesignFound = true;
+        } catch (err) {
+          console.log(err.message);
+        }
+        if (codesignFound) {
+          execSync(`codesign --sign - ${filename}`);
+          execSync(`codesign --verify ${filename}`);
+        }
+      } else if (process.platform === "win32") {
+        let signtoolFound = false;
+        try {
+          execSync("where signtool");
```

Comment:
Apparently, `signtool` isn't globally accessible on the Windows CI runner:
https://app.circleci.com/pipelines/github/postmanlabs/postject/177/workflows/cfb21c18-92b3-48db-b143-4f2f6d016ed8/jobs/1368/steps?invite=true#step-104-7
```sh
INFO: Could not find files for the given pattern(s).
Command failed: where signtool
```
I can think of 2 solutions:
1. Use Talaria, that contains signtool (that's how I verified locally that codesigning works for the Node.js binary)
2. Use `circleci/microsoft-msix@1.1` as described in https://circleci.com/blog/code-signing-with-runner and https://circleci.com/developer/orbs/orb/circleci/microsoft-msix

Preferences or better options anyone?

Reply:
This tool (and others from the Windows SDK) should be on the PATH by default when running on a Visual Studio Developer Prompt (which I believe CircleCI does by default).

Can you access it from outside of this script? i.e. as a standalone step in the CircleCI definition or by SSHing into the worker?

If so, the problem might be in the way you are running this test file or spawning the specific command (without inheriting the outer Visual Studio Developer Prompt environment or something like that).

Reply:
@jviotti 

> Can you access it from outside of this script? i.e. as a standalone step in the CircleCI definition or by SSHing into the worker?

Doesn't seem to be happening:
```console
Microsoft Windows [Version 10.0.17763.3287]
(c) 2018 Microsoft Corporation. All rights reserved.

C:\Users\circleci>signtool
'signtool' is not recognized as an internal or external command,
operable program or batch file.
```

FWIW, it isn't present globally on my Windows 10 either when I run the regular cmd prompt.

Reply:
> FWIW, it isn't present globally on my Windows 10 either when I run the regular cmd prompt.

Yeah, I don't think that will work. That's why I meant the Visual Studio Command Prompt rather than the default one.

Reply:
Maybe just find and use the full path to `signtool.exe` on CircleCI rather than rely on it being on the PATH?

Reply:
I think that's acceptable for now until we open-source Talaria and then make use of it on this public repo.

Reply:
I have turned this into a macOS specific change for now because this is an improvement already. We can add a code signing test for Windows in a separate PR. PTAL!